Java/src/test/java/com/thealgorithms/datastructures/BinaryTreeTest.java

```diff
@@ -0,0 +1,17 @@
+package com.thealgorithms.datastructures;
```

Comment:
I'm not sure about this, but shouldn't the `BinaryTreeTest.java` file be in the `com.thealgorithms.datastructures.trees` package, to match the `BinaryTree.java` file?

It seems to work on my machine, though, so good enough for me.

Reply:
Yes I can fix